corporate/lib/stripe.py

```diff
@@ -342,6 +352,11 @@ def __init__(self, event_type: AuditLogEventType) -> None:
 
 
 class BillingSession(ABC):
+    @property
+    @abstractmethod
+    def billing_session_url(self) -> str:
```

Comment:
There might be a better property name. This will be the base for the URLs returned by `create_stripe_checkout_session` and is `realm.uri` for the `RealmBillingSession` implementation.

Reply:
I could imagine `billing_session_base_url` being better ... but I think we can leave this as-is for. We might find as we use this more that it's the wrong sort of thing and we instead want a function `get_billing_url("upgrade")`, or something using `reverse`? So probably best to start with this simple option until we've got more evidence on what structure will feel nice.